dir.props

```diff
@@ -69,13 +69,15 @@
     <DebugSymbols Condition="'$(DebugSymbols)' == ''">true</DebugSymbols>
     <Optimize Condition="'$(Optimize)' == ''">false</Optimize>
     <DebugType Condition="'$(DebugType)' == ''">full</DebugType>
-    <DefineConstants>$(DefineConstants);DEBUG;TRACE</DefineConstants>
+    <DefineConstants Condition="'$(MSBuildProjectExtension)' != '.vbproj'">$(DefineConstants);DEBUG;TRACE</DefineConstants>
```

Comment:
@weshaggard I am not sure what you mean by saying: "Any VB projects should choose to be semi-colon or comma delimited as I think either would still work." VB compiler doesn't accept semi-colon delimited defines. Replacing semi-colons with commas is error prone because semi-colons could be within values.